### ISSUE TITLE
[hi] Localize /en/docs/reference/glossary/admission-controller.md

### ISSUE THREAD

**Issue by rolfedh** [kubernetes/website#35935]
# Prerequisites: 
- If this is your first contribution, visit [Contribute to K8s docs](https://kubernetes.io/docs/contribute/) for information on how to get started.
- Review a few [examples of localized glossary entries](https://github.com/kubernetes/website/issues/35600#issuecomment-1214215859) to see which content you change and which you live unchanged.

# Procedure:
1. Check out `dev-1.22-hi.1` and create a working branch called `localize-admission-controller.md`.
1. Create a file called `content/hi/docs/reference/glossary/admission-controller.md`.
1. Copy the content of https://raw.githubusercontent.com/kubernetes/website/main/content/en/docs/reference/glossary/admission-controller.md to the file you created.
1. Localize the content.
1. Push your change and create a pull request.

Completing this feature issue fixes part of https://github.com/kubernetes/website/issues/35600

**Comment by khannakshat7** [comment]
Hey @rolfedh, I would like to work on this issue

**Comment by rolfedh** [comment]
> Hey @rolfedh, I would like to work on this issue

Thank you, [khannakshat7](https://github.com/khannakshat7). I will update the instructions telling volunteers to use `/assign <github_id>` when they want to claim an issue like this one. There will be plenty of new issues like this one coming up soon. I look forward to working with you!

**Comment by khannakshat7** [comment]
Absolutely, Looking forward working with you. Thank You @rolfedh for the info and helping me out with this.

**Comment by GitBatra** [comment]
Hey @rolfedh, I would like to work on this issue
/assign

**Comment by rolfedh** [comment]
Hi @GitBatra. Please connect with @bishal7679 about this. Looking above, I see what looks like an open PR waiting to be merged. I also recommend connecting with the team leads on #kubernetes-docs-hi Slack channel, here: https://kubernetes.slack.com/archives/CJ14B9BDJ

**Comment by vaibhav2107** [comment]
@freddie400 
The PR has already been raised to resolve this issue

**Comment by varunseth5788** [comment]
i would like to contribute to this issue.
or need any other help.

**Comment by rolfedh** [comment]
Hi @Shubhdeep02, if you review the PR history, it looks like @amolmote has an open PR for this issue. I recommend connecting with Amol and others on Kubernetes Slack to coordinate your contributions.

**Comment by divya-mohan0209** [comment]
Removing assignees so as to signpost this is accepting new contributors.

**Comment by rolfedh** [comment]
Obsolete
